src/services/blockchain-service.ts

```diff
@@ -54,6 +59,8 @@ export default class BlockchainService implements Contextual {
 
     const txReceipt: TransactionReceipt = await this.provider.waitForTransaction(txResponse.hash);
     const block: Block = await this.provider.getBlock(txReceipt.blockHash);
-    return new Transaction(txResponse.chainId, txReceipt.transactionHash, txReceipt.blockNumber, block.timestamp);
+
+    const caip2ChainId = 'ethereum:eip155-' + txResponse.chainId;
```

Comment:
The correct format should be `'eip155:' + txResponse.chainId`.

Reply:
Fixed.